doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,51 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra                                      > NEW LINE
+
+sudo kdb mount hello.js /hello json
+kdb set user/hello/hello/world "Hello World"
+kdb get user/hello/hello/world
+kdb file user/hello/hello/world
+vi /home/elektra/.config/hello.js
+# Edit file to say "Hello World from File!"
+kdb get user/hello/hello/world
+kdb editor user/hello ini
```

Comment:
Before doing this, write that we recommend to only use `kdb editor` as it will validate configuration?